tests/examples/relationship_PASS.json

```diff
@@ -0,0 +1,85 @@
+{
+    "@context": {
+        "Action": "action:Action",
+        "ArchiveFile": "observable:ArchiveFile",
+        "File": "observable:File",
+        "FileFacet": "observable:FileFacet",
+        "ObservableRelationship": "observable:ObservableRelationship",
+        "Relationship": "core:Relationship",
+        "action": "https://unifiedcyberontology.org/ontology/uco/action#",
+        "core": "https://unifiedcyberontology.org/ontology/uco/core#",
+        "fileName": "observable:fileName",
+        "hasFacet": "core:hasFacet",
+        "isDirectional": "core:isDirectional",
+        "kb": "http://example.org/kb/",
+        "kindOfRelationship": "core:kindOfRelationship",
+        "observable": "https://unifiedcyberontology.org/ontology/uco/observable#",
+        "source": {
+            "@id": "core:source",
+            "@type": "@id"
+        },
+        "target": {
+            "@id": "core:target",
+            "@type": "@id"
+        },
+        "vocabulary": "https://unifiedcyberontology.org/ontology/uco/vocabulary#",
+        "xsd": "http://www.w3.org/2001/XMLSchema#"
+    },
+    "@graph": [
+
+        {
+            "@id": "kb:relationship1",
+            "@type": "Relationship",
+            "isDirectional": true,
+            "kindOfRelationship": "CUSTOM_Influenced",
+            "source": "kb:file1",
+            "target": "kb:zipfile1"
+        },
+        {
+            "@id": "kb:relationship2",
+            "@type": "ObservableRelationship",
+            "isDirectional": true,
+            "kindOfRelationship": {
```

Comment:
@b0bkaT , this is the part that needs to work with context dictionaries.

Reply:
We want line 42 to read:

```json
"kindOfRelationship": "Contained_Within"
```

and the `ObservableObjectRelationshipVocab` should carry over to the Turtle translation.  `make check` will fail if it doesn't convert.